src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -633,16 +519,12 @@ bool emberAfColorControlClusterMoveToHueCallback(chip::app::Command * commandObj
     return true;
 }
 
-bool emberAfColorControlClusterMoveToSaturationCallback(chip::app::Command * commandObj, uint8_t saturation,
-                                                        uint16_t transitionTime, uint8_t optionsMask, uint8_t optionsOverride)
+static bool moveToHueAndSaturation(uint16_t hue, uint8_t saturation, uint16_t transitionTime, uint8_t optionsMask,
```

Comment:
Diff might have been cleaner (and easier to review) if the order had been kept the same (as in, if this were up where `emberAfColorControlClusterMoveToHueAndSaturationCallback` used to be).  Not a big deal, I guess.